python-lib/utils/authentification.py

```diff
@@ -0,0 +1,21 @@
+import requests
+
+
+def get_headers(config, plugin_config):
+    
+    #Retrieve token associated with email and password
+    url = "https://auth.kayrros.com/v2/login"
+
+    username = config["preset"]["username"]
+    password = config["preset"]["password"]
+    
+    req = requests.post(url, json={"email": username, "password": password})
+        
+    if req.status_code == 200:
+        token = req.json()['token']
```

Comment:
Make error more explicit.

```suggestion
    try:        
        response = requests.post(url, json={"email": username, "password": password})
        response.raise_for_status()
    except requests.exceptions.RequestException as error:
        logger.exception("Authentication token could not be retrieved because of the following error:\n {}".format(error))
        raise(error)   
    token = response.json()['token']
```